A/AbstractXsdTypes/Versions.toml

```diff
@@ -1,2 +1,2 @@
 ["0.1.0"]
-git-tree-sha1 = "9b0214aa1bdb299d8a65dc6f25678322f35b2795"
+git-tree-sha1 = "210965258757a9257b98c1bd55e3f1e72b852ee2"
```

Comment:
I believe this should be the git tree hash of the subdir.  @ericphanson can you confirm?
```julia-repl
julia> using Pkg

julia> bytes2hex(Pkg.GitTools.tree_hash("AbstractXsdTypes.jl"))
"c5aea50b0e207f546f48b2470ce0786018a8173f"
```

Reply:
yeah, it should be the result of that function called on the subdir itself

Reply:
Didn't take into account the subdir subtlety.
Should be correct now.
```
julia> bytes2hex(Pkg.GitTools.tree_hash("AbstractXsdTypes.jl"))
"e99adf5c628d7c21e88f3bfbfeb39a4e0c1f061b"
```

Reply:
Uhm, on which commit you computed that?
```console
% git log -1 --oneline 
7a644fc (HEAD -> main, origin/main, origin/HEAD) change UUID AbstractXsdTypes
% julia -E 'using Pkg; bytes2hex(Pkg.GitTools.tree_hash("AbstractXsdTypes.jl"))'
"c5aea50b0e207f546f48b2470ce0786018a8173f"
```

Reply:
The latest on main:
7a644fc (HEAD -> main, origin/main, origin/HEAD) change UUID AbstractXsdTypes

Reply:
A tip is to try a registration with LocalRegistry (`commit=false` is a good idea, and you may need to explicitly point to a clone of General) and the chances that you get a good tree hash are good. Anyway, LocalRegistry finds the tree hash with
```
/tmp/XmlStructTools.jl$ git rev-parse HEAD:AbstractXsdTypes.jl
c5aea50b0e207f546f48b2470ce0786018a8173f
```
which agrees with Mose's result above.

Reply:
Now I do get the same result:
```
$ git rev-parse HEAD:AbstractXsdTypes.jl
c5aea50b0e207f546f48b2470ce0786018a8173f
```